ImplementationGuide/markdown/ISiKundSMART/Schritt1RegistrierungClient.md

```diff
@@ -1,12 +1,8 @@
 # Schritt 1: Registrierung eines SMART Clients mit dem EHR
 
----
-### Informativ
----
+Bevor ein Client eine EHR Launch Sequence oder Standalone Launch Sequence ausführen kann, muss der Client beim Autorisierungsserver des EHR registriert werden. Per Abschnitt [SMART App Launch - 2.0.5 - Register App with EHR](https://hl7.org/fhir/smart-app-launch/STU2.1/app-launch.html#step-1-register) ist ihm freigestellt, wie diese Registrierung durchgeführt wird.
```

Comment:
question: sollen wir den Begriff EHR eventuell zugunsten vom generischen Ressourcenserver vermeiden? Es sei denn, wir nehmen in der gesamten Spec explizit KIS-Systeme als Ressourcenserver in Fokus - wenn dann sollten wir das auch explizit erwähnen.